### PR TITLE
Support for additional security access token

### PR DESCRIPTION
**Pull request by francesco-laricchia**
I needed to set up an "access token", as described in Expo official documentation here: https://docs.expo.io/push-notifications/sending-notifications/#additional-security

Since I couldn't find a way to have it with this library's original code, I have overridden Expo class with my own. These are the changes I did.

### CODE REVIEW

File: lib/Expo.php [hunk 1]
```diff
@@ -73,6 +78,13 @@ public function unsubscribe($interest, $token = null)
     {
         return $this->registrar->removeInterest($interest, $token);
     }
+    
+    /**
+     * @param string|null $accessToken
+     */
+    public function setAccessToken(?string $accessToken = null) {
```
Comment (Alymosul) [inline lib/Expo.php]: Could you remove `?` from the method signature cause I believe that doesn't work with php7.0 which is still supported in the package 🤷‍♂️

File: lib/Expo.php [hunk 2]
```diff
@@ -148,12 +160,18 @@ private function prepareCurl()
     {
         $ch = $this->getCurl();
 
+        $headers = [
+                'accept: application/json',
+                'content-type: application/json',
+        ];
+
+        if ($this->accessToken) {
+            $headers []= sprintf('authorization: bearer %1$s', $this->accessToken);
```
Comment (Alymosul) [inline lib/Expo.php]: Just minor thing `$headers[] = sprintf('Authorization: Bearer %s', $this->accessToken);`

**Review by Alymosul — CHANGES_REQUESTED** [review]
Thank you :) Just a couple of changes needed!

File: lib/Expo.php (hunk 1 above)
Comment (francesco-laricchia) [reply lib/Expo.php]: Yeah, you're right :)

**Review by Alymosul — APPROVED** [review]

**Comment by Alymosul** [comment]
Thanks!

**Pull request merged**